packages/client/src/schedule-types.ts

```diff
@@ -764,13 +764,13 @@ export const MONTHS = [
 ] as const;
 
 /** @experimental */
-export type Month = typeof MONTHS[number];
+export type Month = (typeof MONTHS)[number];
```

Comment:
I don't understand why we have these extra parenthesis added everywhere. have we bumped prettier's version along the way?

Reply:
yeah, I nuked package-lock and node_modules